app/controllers/reports_controller.rb

```diff
@@ -15,163 +15,7 @@ class ReportsController < ApplicationController
   # Calculations in this method must be done reasonably performantly to avoid server timeouts
   # This could eventually be converted to individual REST endpoints for each stat group
   def index
```

Comment:
If we have indeed deprecated this, should we remove it altogether?

Reply:
ah, it's now restored thanks to a revised plotly import